GenderPayGap.WebUI/Models/AccountCreation/CreateUserAccountViewModel.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.Web.Mvc;
+using GovUkDesignSystem;
+using GovUkDesignSystem.Attributes;
+using GovUkDesignSystem.Attributes.ValidationAttributes;
+
+namespace GenderPayGap.WebUI.Models.AccountCreation
+{
+    public class CreateUserAccountViewModel : GovUkViewModel
+    {
+        [GovUkValidateRequired(
+            ErrorMessageIfMissing = "Please enter an email address."
+        )]
```

Comment:
Some of these attributes might look nicer on one line - up to you

Reply:
I agree. Thanks